hangman/lib/hangman/tool.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Hangman.Tool do
+  
+  #List.duplicate/2
+  def get_letters(0),    do: []
+  def get_letters(n),    do: ["_"] ++  get_letters(n-1)
+
+  defmacro is_a_letter(guess) do
+    quote do: is_bitstring(unquote(guess)) and String.length(unquote(guess)) == 1
+  end
+
+  def combine(list_a, _, 0), do: list_a
+
+  def combine(list_a, list_b, n) do
```

Comment:
Again, this is a brute force approach that is a lot harder to write and read than simply using [head|tail] to parse a list.